01week/rockPaperScissors.js

```diff
@@ -8,16 +8,32 @@ const rl = readline.createInterface({
 });
 
 
-function rockPaperScissors(hand1, hand2) {
 
-  // Write code here
+//check for tie, if tie return tie, if not check for win.
 
+function rockPaperScissors(a, b) {
+  const hand1 = a.trim().toLowerCase();
+  const hand2 = b.trim().toLowerCase();
+  if (hand1 === hand2) {
+    return "It's a tie!";
+
+// Make conditional statement for Hand1 to win.
+
+  } else if (hand1 === 'rock' && hand2 === 'scissors' || hand1 === 'paper' && hand2 === 'rock' || hand1 === 'scissors' && hand2 === 'paper') {
+    return "Hand one wins!";
+
+// Make conditional statement for Hand2 to win.
+
+  } else if (hand1 === 'scissors' && hand2 === 'rock' || hand1 === 'paper' && hand2 === 'scissors' || hand1 === 'rock' && hand2 === 'paper') {
```

Comment:
This is redundant. What is away to not have to check all of the combinations again?